notebooks/402-pose-estimation-webcam/402-pose-estimation.ipynb

```diff
@@ -0,0 +1,495 @@
+    "from numpy.lib.stride_tricks import as_strided\n",
+    "from openvino import inference_engine as ie\n",
+    "\n",
+    "from decoder import OpenPoseDecoder\n",
+    "from player import VideoPlayer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download model\n",
+    "\n",
+    "We use `omz_downloader`, which is a command line tool from the `openvino-dev` package. `omz_downloader` automatically creates a directory structure and downloads the selected model.\n",
+    "\n",
+    "If you want to download another model, please change the model name and precision. *Note: This will require a different pose decoder*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# model name as named in Open Model Zoo\n",
+    "model_name = \"human-pose-estimation-0001\"\n",
+    "# selected precision (FP32, FP16, FP16-INT8)\n",
+    "precision = \"FP16-INT8\"\n",
+    "\n",
+    "model_path = f\"model/intel/{model_name}/{precision}/{model_name}.xml\"\n",
+    "model_weights_path = f\"model/intel/{model_name}/{precision}/{model_name}.bin\"\n",
+    "\n",
+    "if not os.path.exists(model_path):\n",
+    "    download_command = f\"omz_downloader \" \\\n",
+    "                       f\"--name {model_name} \" \\\n",
+    "                       f\"--precision {precision} \" \\\n",
+    "                       f\"--output_dir {base_model_dir}\"\n",
+    "    ! $download_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Load model\n",
+    "\n",
+    "Downloaded models are located in a fixed structure, which indicates vendor, model name and precision.\n",
+    "\n",
+    "Only a few lines of code are required to run the model. First, we create an Inference Engine. Then we read the network architecture and model weights from the .bin and .xml files to load onto the desired device."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# initialize inference engine\n",
+    "ie_core = ie.IECore()\n",
+    "# read the network and corresponding weights from file\n",
+    "net = ie_core.read_network(model=model_path, weights=model_weights_path)\n",
+    "# load the model on the CPU (you can use GPU or MYRIAD as well)\n",
+    "exec_net = ie_core.load_network(net, \"CPU\")\n",
+    "\n",
+    "# get input and output names of nodes\n",
+    "input_key = list(exec_net.input_info)[0]\n",
+    "output_keys = list(exec_net.outputs.keys())\n",
+    "\n",
+    "# get input size\n",
+    "batch_size, channels, height, width = net.inputs[input_key].shape"
```

Comment:
This shows a deprecation warning. Can you rewrite this with the new syntax that does not show this?